app/src/main/java/org/fossasia/susi/ai/helper/Utils.kt

```diff
@@ -58,4 +57,13 @@ object Utils {
         return result.toString()
     }
 
+    fun truncateEmailAtEnd(email: String?): String? {
+        if (!email.isNullOrEmpty()) {
+            val truncateAt = email?.indexOf('@')
+            if (truncateAt is Int && truncateAt != -1) {
```

Comment:
`truncateAt is Int` is not required?

Reply:
As email is nullable, ```truncateAt``` is being called upon a nullable String object. Later when I have used ```truncateAt.plus(1)``` for that we strictly require ```Int``` but ```truncateAt``` is of type ```Int?```
Hence, I have used ```truncateAt is Int``` when the email is not null or empty.
Could this be improved?

Reply:
`truncateAt != null and ...`